sparkly/utils.py

```diff
@@ -15,16 +15,21 @@
 #
 
 import os
+from collections import OrderedDict
 
 try:
     from kafka import SimpleClient
     from kafka.structs import OffsetRequestPayload
 except ImportError:
     pass
+from pyspark.sql.types import (StructType, StringType, LongType, IntegerType,
```

Comment:
Minor style comment but in general, I very much prefer the style `from pyspark.sql import types as T` (and similarly `functions as F`) and then using `T.IntegerType` etc in the code as needed. This is the style suggested at the spark docs and also you don't need to have multiline imports for all the small things you might need